packages/integration-tests/cmr.js

```diff
@@ -85,31 +90,28 @@ async function conceptExists(cmrLink) {
   return response.body.feed.entry.length > 0;
 }
 
+const waitForCmrToBeConsistent = () => sleep(ONE_SECOND);
+
 /**
  * Checks for granule in CMR until it get the desired outcome or hits
  * the number of retries.
  *
  * @param {string} cmrLink - url for granule in CMR
  * @param {boolean} expectation - whether concept should exist (true) or not (false)
- * @param {string} [retries=3] - number of remaining tries
- * @param {number} [interval=2000] - time (in ms) to wait between tries
  * @returns {Promise<undefined>}
  * @throws {TimeoutError} - throws error when timeout is reached
  */
-async function waitForConceptExistsOutcome(cmrLink, expectation, retries = 3, interval = 2000) {
+async function waitForConceptExistsOutcome(cmrLink, expectation) {
   try {
     await pWaitFor(
       async () => (await conceptExists(cmrLink)) === expectation,
-      {
-        interval,
-        timeout: interval * retries
-      }
+      { interval: THREE_SECONDS, timeout: ONE_MINUTE }
     );
 
-    console.log(`waitForConceptExistsOutcome() succeeded. conceptExists('${cmrLink}') === ${expectation}`);
+    await waitForCmrToBeConsistent();
```

Comment:
This is obviously well named, but can we get a comment explaining why it's necessary?

Reply:
Comment added on line 93.